modules/ROOT/nav.adoc

```diff
@@ -49,7 +49,12 @@
     *** xref:guides:delete.adoc[]
   ** xref:guides:transactions.adoc[]
   ** xref:guides:import.adoc[]
-  ** xref:guides:udfs-for-query/udfs-for-query-introduction.adoc[]
+  ** xref:guides:javascript-udfs/introduction.adoc[]
+    *** xref:guides:javascript-udfs/creating-javascript-udf.adoc[]
+    *** xref:guides:javascript-udfs/calling-javascript-udf.adoc[]
+    *** xref:guides:javascript-udfs/calling-sqlpp-from-javascript.adoc[]
+    *** xref:guides:javascript-udfs/data-handling-javascript-udf.adoc[]
+    *** xref:guides:javascript-udfs/handling-errors-javascript-udf.adoc[]
```

Comment:
I'm going to leave this one I think.

It's better to have the guides in separate folders, especially when you have quite a few images and example code running through them It just makes it a lot easier to find the bits you're working on.

Reply:
I'm going to push back on this a little bit, because IME moving existing files to different locations always leads to broken links and frustrating unforeseen problems. I take your point that it's easier for an author to work on a single isolated set of files. But my (general) observation is that we're never working on a single set of files: they fit into a wider context, with shared images, shared examples, and shared partials, etc. Of course that's harder for the author, because you have to learn that wider context and keep it in your head. But consistency helps. So sorry to be a pain, but I'm going to recommend that you put the UDF guide pages, images, and partials into the same structure as the other existing guides. Examples can or course stay separate, because there are such a lot of them, and the other guide examples are already split up into subdirectories.

Please note the top-level `javascript-udf` module is a great idea and can stay exactly where it is. I'll decide whether it belongs under Develop or Reference later.